src/components/views/elements/AppPermission.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import url from 'url';
+
+export default class AppPermission extends React.Component {
+    constructor(props) {
+        super(props);
+
+        const curlBase = this.getCurlBase();
+        this.state = {
+            curlBase: curlBase,
+        };
+    }
+
+    // Return string representation of content URL without query parameters
+    getCurlBase() {
+        const wurl = url.parse(this.props.url);
+        let curl;
+        let curlString;
+
+        const searchParams = new URLSearchParams(wurl.search);
+
+        if(searchParams && searchParams.get('url')) {
```

Comment:
Thanks for the discussion on this Kegsay. I've fixed as suggested - 5f8e86197fe9677f5beee125da25921887a601b6